packages/celotool/src/lib/cluster.ts

```diff
@@ -137,6 +138,8 @@ export async function setupCluster(celoEnv: string, createdCluster: boolean) {
 
   await installCertManagerAndNginx(celoEnv)
 
+  await installNFSServerProvisioner(celoEnv)
```

Comment:
I'm thinking now if this would be more appropriate to move to blockscout install/upgrade command/logic instead of cluster setup. My argument for moving is not installing the NFS chart on clusters that won't run blockscout (like oracle's or forno's), bc right now the NFS dependency is very specific to blockscout.
My impression is moving this function invocation to the blosckout deployment commands should work as it is now.